pom.xml

```diff
@@ -98,6 +98,34 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.3</version>
+        <configuration>
+          <compilerId>javac-with-errorprone</compilerId>
+          <forceJavacCompilerUse>true</forceJavacCompilerUse>
+          <!-- maven-compiler-plugin defaults to targeting Java 5, but our
+               javac only supports >=6 -->
+          <source>8</source>
```

Comment:
This version is linked to  the property 'maven.compiler.source', right? Maybe create a new property that is e.g. used by both?

Reply:
Good point! Those properties actually directly set the values for the plug-in (I think :sweat_smile: ), so I just removed both the `source` and `target` property setters there to see if they would use those: still works!